docs/SuccessFactors-batchsource.md

```diff
@@ -19,8 +19,21 @@ annotating metadata, etc.
 **Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.  
 **Connection:** Name of the connection to use. Entity Names information will be provided by the connection.
 You also can use the macro function ${conn(connection-name)}.  
+**Authentication Type:** Type of authentication used to submit request. OAuth2, Basic Authentication types are available.  
```

Comment:
Rewrite it to: `Authentication type used to submit request. Supported types are Basic & OAuth 2.0. Default is Basic Authentication.`